### PR TITLE
yashim/hotfix: deriv.me login deriv app id

### PR DESCRIPTION
**Pull request by yashim-deriv**
Co-authored-by: Yashim Wong <yashimwong@gmail.com>

Changes:
-  Add check, if domain is deriv.me, use a different app_id for login

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by sean-binary — DISMISSED** [review]

File: src/common/constants.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ export const dp2p_google_play_url =
 export const gtm_test_domain = 'deriv-com.binary.sx'
 export const livechat_client_id = '66aa088aad5a414484c1fd1fa8a5ace7'
 export const livechat_license_id = 12049137
+export const deriv_app_app_id = domain_url === deriv_me_url ? 1411 : 16929
```
Comment (ShuvoHabib) [inline src/common/constants.js:51]: I have small change request. Let's save the app_id's in descriptive variable, and use in this line.  And we can use 'deriv_com_url' as primary check.

Comment (yashim-deriv) [reply src/common/constants.js]: I've applied the suggestions~ @ShuvoHabib do check if it is clear enough, thanks 😄

**Review by ShuvoHabib — APPROVED** [review]
LGTM

File: src/common/constants.js (hunk 1 above)
Comment (ShuvoHabib) [reply src/common/constants.js]: Perfect, mate 😄

**Review by sean-binary — APPROVED** [review]

**Pull request merged**